src/python/e2e-test-runner/e2e_test_runner/main.py

```diff
@@ -1,32 +1,35 @@
 import os
-import shutil
 import sys
-from typing import Any
+from pathlib import Path
 
 from grapl_common.debugger.vsc_debugger import wait_for_vsc_debugger
 from grapl_common.grapl_logger import get_module_grapl_logger
 
 LOGGER = get_module_grapl_logger()
 
 
-def lambda_handler(event: Any, context: Any) -> None:
+def main() -> None:
     wait_for_vsc_debugger("grapl_e2e_tests")
-    LOGGER.info("staging schemas")
-    if os.path.exists("/tmp/schemas"):
-        shutil.rmtree("/tmp/schemas")
-    os.mkdir("/tmp/schemas")
-    shutil.copyfile("e2e_test_runner/schemas.py", "/tmp/schemas/schemas.py")
-    LOGGER.info("staged schemas")
 
     LOGGER.info("executing pytest")
 
     from grapl_tests_common import setup_tests  # import here to limit monkeypatch
 
+    _cd_to_test_directory()
     result = setup_tests.exec_pytest()
 
     LOGGER.info(f"tests completed with status code {result}")
 
-    if result == 0:
-        return  # Lambda runtime will flip out if sys.exit(0) -_-
-    else:
-        sys.exit(result)
+    sys.exit(result)
+
+
+def _cd_to_test_directory() -> None:
```

Comment:
Not saying this is *the right way* to do it, but it works for now and gets us out of weird awslambda stuff.